CONTRIBUTING.md

```diff
@@ -82,4 +82,4 @@ The code for the documentation and the website are located in the meanjs/mean re
 
 For those who will be generating releases:
 * Don't forget to update the changelog!
-  * `conventional-changelog -r 1`
+  * `conventional-changelog -r 1 | cat > CHANGELOG.md`
```

Comment:
@codydaig since that command appends to the file it will cause later versions to appear at the bottom of the file, right?

If so, then you need to figure out how to prepend to the file, something like one of these: http://www.commandlinefu.com/commands/view/14354/prepend-text-to-a-file